stp_zmq/zstack.py

```diff
@@ -802,6 +804,26 @@ def transmit(self, msg, uid, timeout=None, serialized=False):
         return False, err_str
 
     def transmitThroughListener(self, msg, ident) -> Tuple[bool, Optional[str]]:
+        self._waiting_messages.setdefault(ident, [])
+        self._waiting_messages[ident].append(msg)
+        result = True
+        error_msg = None
+        for current_msg in list(self._waiting_messages[ident]):
+            result, error_msg, need_to_resend = self._transmit_one_msg_through_listener(current_msg,
+                                                                                        ident)
+            if not need_to_resend:
+                self._waiting_messages[ident].remove(current_msg)
```

Comment:
Are we going to remove `ident` from `_waiting_messages` if there are no more messages there?